app/scripts/controllers/network/network-controller.js

```diff
@@ -224,42 +239,48 @@ export default class NetworkController extends EventEmitter {
     }
   }
 
-  setRpcTarget(rpcUrl, chainId, ticker = 'ETH', nickname = '', rpcPrefs) {
```

Comment:
I see that you've removed `setRpcTarget` and replaced it with `setActiveNetwork`. I figured we might remove this in another PR, but that's okay. If we're going to remove `setRpcTarget`, though, then do we need `setProviderType` anymore? They both seem to do similar things.

Reply:
`setProviderType` is for preloaded infura powered networks while `setActiveNetwork` is for user added networks. I think we probably can/should consolidate these, but it would require another suit of changes where this method is used and at this point I'm thinking the changes in this PR have grown sufficiently such that perhaps it makes more sense to do that in a separate PR. I could see an argument either way though.

Reply:
Oh — so preloaded networks won't be in network configurations, they'll live outside?

Reply:
Correct. As of right now that is the way it works.

Reply:
Ah. Okay!